citi-onboarding/DADe-onepage
# feat(feature/members-mobile): create members mobile

## What I did

- Create members mobile version
- Add rounded images for all members versions
- Change 'texto_roxo' to 'texto_branco_ in banner model
- Fix disappearing Participate button in mobile banner

| Mockup | Real |
|--------|------|
|![Captura de Tela_select-area_20210623115107](https://user-images.githubusercontent.com/61971582/123119073-955b6d80-d419-11eb-8d5f-e659e73ec7f4.png)|![Captura de Tela_select-area_20210623115930](https://user-images.githubusercontent.com/61971582/123120439-afe21680-d41a-11eb-8000-4687dcf44e78.png)|


| Mockup | Real |
|--------|------|
![Captura de Tela_select-area_20210623115120](https://user-images.githubusercontent.com/61971582/123119136-a310f300-d419-11eb-922f-b0b4784816f6.png)|![membros_real](https://user-images.githubusercontent.com/61971582/123119153-a6a47a00-d419-11eb-9e61-5f514de42525.png)|

## How to test

1. Open the terminal and go to the folder of this project

2. In terminal:
   1. `git checkout develop`
   2. `git pull`
   3. `git checkout feature/members-mobile`
   4. `git pull origin feature/members-mobile`
   5. `npm install`
   6. `cd server/
   7. `node index.js`
   8. Open a new terminal
   9. `cd client/`
   10. `npm start`

3. In your browser go to http://localhost:3000 (should already be opened by `npm start`)

4. Check if the component looks just like the mockup in different screen sizes, and in both the mobile and desktop versions. Check if the slick is working as it should.

5. Check if the banner button is appearing in the mobile version (bug fixed)

6. Read the changed files section on GitHub 

7. Test the things that you think are worth testing, even the ones that are not in this description
